app/src/main/res/layout/activity_main.xml

```diff
@@ -1,18 +1,70 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    tools:context=".MainActivity">
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    android:layout_margin="@dimen/layout_margin"
+    android:layout_marginStart="@dimen/layout_left_margin"
+    android:layout_marginEnd="@dimen/layout_right_margin"
```

Comment:
Why do you need `layout_margin`, and `layout_marginStart`, `layout_marginEnd`?
Either use only `layout_margin` to give margin for all 4 sides (start, end, top, bottom) or give individual margins like `layout_marginStart`, `layout_marginEnd`, `layout_marginTop` and `layout_marginBottom`.